internal/terminal/table.go

```diff
@@ -14,8 +14,12 @@ const (
 
 	// gutter is the gap between table columns
 	gutter = "  "
-	// indent is the indentation used
-	indent = "  "
+)
+
+// set of supported spacing options
+const (
+	// Indent is the indentation used
```

Comment:
[optional] to me, the `// set of ...` comment here suffices for docstrings and you could remove this one here

i would also probably move `gutter` down into this block (rather than above), just since its also a "spacing option" (supported =/= exported, i think we're fine here by the letter of the law 🙃 )